app/resources/en/translation/applicant/address.json

```diff
@@ -15,7 +15,7 @@
   "errors": {
     "addressLine1": {
       "required": {
-        "summary": "Enter the first line of the address",
+        "summary": "You haven&rsquo;t entered the first line of the address",
```

Comment:
Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence